### PR TITLE
Add dishes to order

### PR DESCRIPTION
**Pull request by flamesoft**
PT story: https://www.pivotaltracker.com/story/show/121662745
- Display "Add to cart" button for every dish on the dish details page
- Display message for cart status on the same page, e.g. "You have added 1 dish", on the header

### CODE REVIEW

File: features/add_to_cart.feature [hunk 1]
```diff
@@ -28,4 +28,5 @@ Feature: As a visitor
       Given I am logged-in as "Calle"
       And I am on the "dish list page" for menu "Breakfast"
       And I click "Add to cart" for "Dimsun"
-      #hen show me the page
+      Then show me the page
```
Comment (tochman) [inline features/add_to_cart.feature]: This needs to go ;--)

**Comment by tochman** [comment]
The problem I see here is that we don't have tests for adding multiple dishes to an order. I mentioned in the session that we will need to condition the creation of an Order IF there is no order already created. If you plan to extract that functionality to a separate story, that needs to be created in PT.

File: features/add_to_cart.feature (hunk 1 above)
Comment (flamesoft) [inline features/add_to_cart.feature]: Forgot this. I will remove this.

**Comment by tochman** [comment]
@diraulo can we have your take on this?

**Pull request merged**